src/containers/GalacticPropertiesComboContainer.jsx

```diff
@@ -22,6 +22,7 @@ class GalacticPropertiesComboContainer extends React.PureComponent {
         name: 'Brightness vs Distance',
         color: 'rgb(254, 216, 40)',
         options: {
+          svgShape: 'triangle',
```

Comment:
Would using the `ChartSwitcherContainer` allow you to adopt your `svgShapes` options conventions more consistently here?  Feels of to go through the trouble of setting up this system of options that get passed down to the widgets and then have the widget that inspired this feature in the first place balk that new convention.

Reply:
How do you believe the ChartSwiterContainer would be a good fit? I'm not sure I see your meaning or intended use?

Reply:
Was thinking you could then be modeling your charts in the pageData instead of hardcoded in this part of the container `Class`.  Which in turn would mean you would be impacting the charts' shapes in the same manner you've done elsewhere: options on the widget in the pagedata json.  But I don't have a sense if there's other bespoke stuff going on in `ComboContainer` that would make the jump to the more generalized `ChartSwitcherContainer` prohibitively difficult.

Reply:
Will pull of this hardcoded data and place into the json.

Reply:
I look at extracting this work into the json, but was not able to figure out how to do so. Maybe I'm unaware of an options prop that takes an array of options, like an "options set" or something in the json query. I think that's why I hardcoded it in the first place.

For now, I've added `svgShapes: ["shapeName"]` into the json and updated the `this.properties` in the `GalacticPropertiesContainer` to allow for that customization.